packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -169,11 +204,17 @@ class TextPainter {
   /// relative a typical line of text.
   ///
   /// Obtaining this value does not require calling [layout].
+  ///
+  /// The style of the [text] property is used to determine the font settings
+  /// that contribute to the [preferredLineHeight]. If [text] is null or if it
+  /// specifies no styles, the default [TextStyle] values are used (a 10 pixel
+  /// sans-serif font).
   double get preferredLineHeight {
-    assert(text != null);
     if (_layoutTemplate == null) {
-      final ui.ParagraphBuilder builder = new ui.ParagraphBuilder(_createParagraphStyle());
-      if (text.style != null)
+      final ui.ParagraphBuilder builder = new ui.ParagraphBuilder(
+        _createParagraphStyle(TextDirection.ltr),
+      ); // direction doesn't matter, text is just a zero width space
```

Comment:
Should we use RTL if it doesn't matter?  That's more likely to catch errors given that we do more manual testing in LTR.

Reply:
sure, why not. done.